README.md

```diff
@@ -1 +1,366 @@
+| OP_CORS_PATH_PATTERN | Allowed origin URL path. This path will be the same for all dev, test, prod profiles | "/**" In application.properties this value is already set so this environment variable is OPTIONAL|
+| OP_WALLET_CONNECTION | If the connection between the application and the database is via wallet. For more information please read the wallet database connection below | true,false  |
+
+For reCAPTCHA the solution supports the google reCAPTCHA v2. In order to use the reCAPTCHA you will need your own recaptcha site configuration and include the server key in the ./Dockerfile or in the ./application/main/resources/application.properties
+
+**regular database connection**
+
+Note that the containerize/extra-jars folder contains the Oracle specific connection jars. In case you need other jar(s) for database connection you have to include that in the Dockerfile.
+
+If you want to go with the regular connection you need the following settings at build time:
+* update the following connection string according to your database settings
+"jdbc:oracle:thin:@//service:port/server_name" (service name is the name of the service,port the database connection port, server name is the server name where the database is located.) and set the "OP_DATASOURCE_URL" to that
+* set the database credentials for "OP_DATASOURCE_USERNAME" and "OP_DATASOURCE_PW"
+* set the "OP_WALLET_CONNECTION" to "false"
+
+**wallet database connection**
+
+Note that the containerize/extra-jars folder contains the Oracle specific connection jars. In case you need other jar(s) for database connection you have to include that in the Dockerfile.
+
+Some databases for example the Oracle Autonomous database requires a database wallet for the connection. The wallet is a zip file containing the configurations for SSL connection. 
+
+If you want to use wallet you need the following settings at build time:
+* set the "OP_WALLET_CONNECTION" varibale to true and during the start 
+* set the "OP_WALLET_PATH" where you want to have the settings extracted
+* copy the wallet under the containerize/db_wallet folder with the following name: Wallet_DB.zip
+* update the following connection string according to your database settings
+"jdbc:oracle:thin:@opendays_high?TNS_ADMIN=/opt/wallet" (opendays_high is the database service which we connect, TNS_ADMIN is the location where we extract the wallet.) and set the "OP_DATASOURCE_URL" to that
+* set the database credentials for "OP_DATASOURCE_USERNAME" and "OP_DATASOURCE_PW"
+
+**Containerize folder**
+
+This folder contains helper files for the image build.
+* **extra-jars**: The jar files for creating the connection to an Oracle Autonomous database
+* **startup_scripts**: The startup scripts for application building and hosting.
+
+# How to use the container
+
+To run the image you need to have [Docker installed](https://docs.docker.com/install/).
+
+Once your environemnt is ready you update the settings in the ./Dockerfile. 
```

Comment:
./Dockerfile is the Dockerfile in the root folder, like https://gitlab.cern.ch/db/public-events/cern-opendays/registration-application-back-end/blob/master/Dockerfile ?

Reply:
Yes, I have put it under the root folder. I have applied this change from our gitlab version. Also I changed a bit the docker image, because Antonio was not sure if he is going to publish the deployment script, so now it should be independent from that.

Reply:
We discussed, it's in the root folder nothing to do